test/clt-tests/core/test-auto-schema.rec

```diff
@@ -0,0 +1,60 @@
+––– block: ../base/start-searchd –––
+––– input –––
+mysql -h0 -P9306 -e "drop table test"
+––– output –––
+ERROR 1064 (42000) at line 1: DROP TABLE failed: unknown table 'test'
+––– input –––
+curl -X POST "localhost:9308/sql?mode=raw" -d "insert into test(id, f, a, b, j, m, d, v) values (1, 'sample text', 42, 3.1415, '{\"key\": \"value\"}', (1, 2, 3), 1625142600, '[12345678901234, 56789012345678]')"; echo
+––– output –––
+[{"total":1,"error":"","warning":""}]
+––– input –––
+mysql -h0 -P9306 -e "select * from test"
+––– output –––
++------+-------------+------+----------+-----------------+-------+------------+---------------------------------+
+| id   | f           | a    | b        | j               | m     | d          | v                               |
++------+-------------+------+----------+-----------------+-------+------------+---------------------------------+
+|    1 | sample text |   42 | 3.141500 | {"key":"value"} | 1,2,3 | 1625142600 | [12345678901234,56789012345678] |
++------+-------------+------+----------+-----------------+-------+------------+---------------------------------+
+––– input –––
+mysql -h0 -P9306 -e "describe test"
+––– output –––
++-------+--------+----------------+
+| Field | Type   | Properties     |
++-------+--------+----------------+
+| id    | bigint |                |
+| f     | text   | indexed stored |
+| a     | uint   |                |
+| b     | float  |                |
+| j     | json   |                |
+| m     | mva    |                |
+| d     | uint   |                |
+| v     | json   |                |
++-------+--------+----------------+
+––– input –––
+curl -X POST "localhost:9308/sql?mode=raw" -d "insert into test2(id, model, sold, date_added, storage_capacity, price, additional_info, vector) VALUES (1, 'Model_1', 1, 1620590570251, 512, 654.19, '{\"features\": [\"Feature_1\", \"Feature_2\"]}', '[0.1, 0.2, 0.3, 0.4]'), (2, 'Model_2', 1, 1689374130160, 128, 663.70, '{\"features\": [\"Feature_1\", \"Feature_2\"]}', '[0.5, -0.3, 0.9, -0.1]')"; echo
```

Comment:
Does `test2` exist at this moment. I see you did `drop table test`, but didn't do it for `test2`.